pulp_puppet_extensions_admin/pulp_puppet/extensions/admin/pulp_cli.py

```diff
@@ -7,7 +7,7 @@
 
 from pulp_puppet.common import constants
 from pulp_puppet.extensions.admin import structure
-from pulp_puppet.extensions.admin.consumer import bind, content
+from pulp_puppet.extensions.admin.consumer import bind
```

Comment:
We will still support binding, but it will only serve as a relationship between a consumer and a publisher. We need to know those relationships for applicability calculation;.